src/common/config.go

```diff
@@ -75,6 +75,8 @@ type FeaturesConfig struct {
 	Import_cache        string `json:"import_cache"`
 	Downsize_paused_mem bool   `json:"downsize_paused_mem"`
 	Enable_seccomp      bool   `json:"enable_seccomp"`
+	Warmup              bool   `json:"warmup"`
```

Comment:
Warmup could mean a lot of things -- probably use a longer name referencing "import cache" or zygotes" so folks know what this is about.

Reply:
On that note, should we be pulling all zygote-related options together into the same struct?  It's starting to get a little scattered with Import_cache_tree, Import_cache, Warmup...